contributing.md

```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
 **Prerequisites**
 -Node.js version v18.17.1
 
-**Here are some steps you can follow:**
-- Fork the project to your own GitHub account.
-- Clone the forked project to your local machine.
-- Run ```npm install``` command
-- Make changes to the code or documentation.
-- Test the changes you have made using ```npm run dev``` or ```npm run dev -- --host```.
-- Commit your changes and push them to your forked repository.
-- Submit a pull request to the main project repository.
+There are some steps that you need to follow before making any contribution in this project, the following steps are important to get start.
+
+### Step 1. Fork the Repository
+ [Fork](https://github.com/ajaynegi45/Library.git) the repo first
+
+### Step 2. Clone the forked Repository 
+    ```
+    git clone https://github.com/<your_github_username>/Library.git
+    ```
+### Step 3. Set Up Remotes 
+- Add a reference (remote) to the original repository:
+    ```
+    git remote add upstream https://github.com/ajaynegi45/Library.git
+    ``` 
+    - Add a reference (remote) to your forked repository:
+```
+git remote add origin https://github.com/<your_user_name>/Library.git
```

Comment:
Align both point and code properly

<img width="1064" alt="Screenshot 2023-10-06 at 8 52 47 PM" src="https://github.com/ajaynegi45/Library/assets/97536703/72f67b04-b4ce-4859-bde4-eb30f869a43e">

Reply:
okay